TradeTrust/tradetrust-website
# fix: bump synpress

## Summary

synpress v3.0.5 started to fail consistently for **all** tests (metamask flow)

## Changes

- Tests:
  - v3.0.5 appears not to be adding local testnet during initial metamask setup
    - without local testnet, all tests will fail
    - bumping to v3.5.1 fixes it
  - tests sometimes fail due to font-awesome load. change to cdn link, with the same version as [tradetrust-ui](https://github.com/TradeTrust/tradetrust-ui-components/blob/4f8ff451c0d8f68a838b711b2b513acee7308d10/src/tailwind.css#L1)
  
![error-font-awesome](https://user-images.githubusercontent.com/4774314/232680611-2ab90d3c-10ca-47c3-82b2-b5bd7df4f6b9.png)

- Housekeep:
  - Remove deploying of local document store, not needed
  - Remove unused variables
  - Remove duplicate import of Ubuntu font
